modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -218,7 +237,7 @@
                ;; meaning this table is partitioned by ingestion time
                ;; so we manually sync the 2 pseudo-columns _PARTITIONTIME AND _PARTITIONDATE
                (and is-partitioned?
-                    (some? (.getTimePartitioning tabledef))
+                    (some? (tabledef->time-partition tabledef))
```

Comment:
potentially being pedantic here, but I _think_ this change isn't covered by tests

Reply:
it should be covered [here](https://github.com/metabase/metabase/blob/79097cfb24c92e01b399d8875f7b63936aaee06f/modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk_test.clj#L412) no?

Reply:
I mean the `change` is not covered. That is a test for normal tables, not materialized views.